/**:
  ros__parameters:
    scan_duration: 0.1
    max_buffer_size: 100
    use_imu: true
    use_odom: true
